### PR TITLE
Add mborsz to sig-scalability config OWNERS.

### PR DESCRIPTION
**Pull request by mborsz**
Add me to sig-scalability config OWNERS.

/assign @wojtek-t 
/cc @krzysied

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/OWNERS [hunk 1]
```diff
@@ -1,3 +1,4 @@
 approvers:
 - shyamjvs
 - wojtek-t
+- mborsz
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/OWNERS]: please sort alphabetically

**Comment by wojtek-t** [comment]
/approve

Approved - feel free to ask someone else for lgtm once comment is resolved.

**Comment by mborsz** [comment]
comment resolved

**Comment by mborsz** [comment]
/assign @BenTheElder 

Benjamin, could you lgtm this change given wojtek-t's comment?

**Pull request merged**